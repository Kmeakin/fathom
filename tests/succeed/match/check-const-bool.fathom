let x : Bool = false;

match x {
    false => 0,
    x => 1,
} : U8
