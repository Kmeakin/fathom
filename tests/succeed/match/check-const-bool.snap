stdout = '''
let x : Bool = false; match x { false => 0, _ => 1 } : U8
'''
stderr = ''
